manual/CHAPTER_CellLib.tex

```diff
@@ -345,9 +358,11 @@ \subsection{Memories}
 This input is \B{WR\_PORTS}*\B{WIDTH} bits wide, containing all data signals for the write ports.
 \end{itemize}
 
-The {\tt techmap} pass can be used to manually map {\tt \$mem} cells to
-specialized memory cells on the target architecture, such as block ram resources
-on an FPGA.
```

Comment:
I've removed this passage because I *think* this isn't actually done by `techmap`? Or else what would be all the custom steps in `synth_*` for.

Reply:
You could do it with `techmap`. But then I wrote the bram mapper which is better for the job in almost all practical cases. (That paragraph is from before the bram mapper existed. I think it's a good choice to remove it.)